pom.xml

```diff
@@ -1897,22 +1904,24 @@
             <validationSet>
               <dir>config</dir>
               <excludes>
-                <exclude>java.header</exclude>
-                <exclude>java-regexp.header</exclude>
-                <exclude>org.eclipse.jdt.core.prefs</exclude>
-                <exclude>intellij-idea-inspections.properties</exclude>
-                <exclude>markdownlint.rb</exclude>
-                <exclude>signatures.txt</exclude>
-                <exclude>signatures-test.txt</exclude>
-                <exclude>codenarc-rules.groovy.txt</exclude>
-                <!-- until https://github.com/checkstyle/checkstyle/issues/11572 -->
-                <exclude>linkcheck-suppressions.txt</exclude>
+                <exclude>.spotless/import.order</exclude>
+                <exclude>.spotless/licence.header</exclude>
                 <exclude>archunit-store/**</exclude>
+                <exclude>codenarc-rules.groovy.txt</exclude>
```

Comment:
this is something i would expect from the sortPom feature but its more a format thats its actually changing the rows like i did by sorting them.